consoles/video_stream.pm

```diff
@@ -0,0 +1,395 @@
+# Copyright © 2021 Marek Marczykowski-Górecki
+# Copyright © 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0-or-later
```

as in https://github.com/os-autoinst/os-autoinst/blob/master/consoles/console.pm#L3